Doc/library/functions.rst

```diff
@@ -1690,7 +1690,7 @@ are always available.  They are listed here in alphabetical order.
 
 
 .. class:: type(object)
-           type(name, bases, dict)
+           type(name, bases, dict, **kwds)
```

Comment:
Personally I like kwds more, but the rest of the document seems to be using ``**kwargs`` or ``**kwarg`` (there's one ``**kws`` too !), so I'd suggest using ``**kwargs`` to keep consistent.

```suggestion
           type(name, bases, dict, **kwargs)
```

Reply:
That was my initial choice, but typeshed and the section on customizing class creation in the docs mostly uses `**kwds` (it also uses `**kwargs` once). Given that reasoning, if you still think `**kwargs` is best I'm happy to change it.

Reply:
Up to you. I think as it stands right now it's okay :).